<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>huetagwriterwindow</class>
 <widget class="QDialog" name="huetagwriterwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>huetagwriterwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <widget class="QLabel" name="markerImageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>710</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
background-color: rgb(66, 66, 66); color : white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>860</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(198, 40, 40); color : white;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveImageButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>860</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(106, 27, 154); color : white;
qproperty-frame : false;</string>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>710</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Huetag ID (1 - 16777215):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="generateMarkerButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>710</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(25, 118, 210); color : white;
qproperty-frame : false;</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>750</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,0,0);</string>
   </property>
   <property name="text">
    <string>Only signed numbers allowed.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Marker ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Main Bits: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="IDLabel_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Parity Bits:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="parityBitsLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>660</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mainBitsLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>00 00 00 00 00 00 00 00 00 00 00 00 00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="markerIdLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,0); color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>10152195</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
